skiko/src/iosMain/kotlin/org/jetbrains/skiko/Actuals.ios.kt

```diff
@@ -15,6 +15,8 @@ internal actual fun ClipboardManager_getText(): String? {
     return UIPasteboard.generalPasteboard.string
 }
 
+internal actual fun ClipboardManager_hasText(): Boolean = UIPasteboard.generalPasteboard.hasStrings()
```

Comment:
Can we just return null in `open fun getText(): String?` if this is true?

Reply:
> But, we can access method generalPasteboard.hasStrings() to get clipboard content. without permission.

Also, wouldn't it more right, if we be checking for permission itself in Compose? Having `hasText` and returning null can mislead users, and hide the error.

Reply:
> if we be checking for permission itself in Compose

it can be provided via Skiko though, as you did. It seems we just need to rename the function to something like `hasPermissions`

Reply:
decided that reading text is overkill and not performant if we need to just check if the clipboard is empty